explore/scripts/get_repos_users.py

```diff
@@ -103,12 +113,22 @@
 print("\nCollective data gathering complete!")
 
 # Combine new data with existing data
-allData["data"] = collective["data"]
+allData["data"] = collectiveExt["data"]
+allDataString = json.dumps(allData, indent=4, sort_keys=True)
+
+# Write output file
+print("\nWriting file '%s'" % (datfilepathExt))
+with open(datfilepathExt, "w") as fileout:
+	fileout.write(allDataString)
+print("Wrote file!")
+
+# Combine new data with existing data
+allData["data"] = collectiveInt["data"]
 allDataString = json.dumps(allData, indent=4, sort_keys=True)
 
 # Write output file
-print("\nWriting file '%s'" % (datfilepath))
-with open(datfilepath, "w") as fileout:
+print("\nWriting file '%s'" % (datfilepathInt))
+with open(datfilepathInt, "w") as fileout:
```

Comment:
Do you need two data writes here? One for int and one for ext?

Reply:
Yup, two different files, two data writes.
datfilepathExt = "../github-data/extUsers.json" (lines 115-123)
datfilepathInt = "../github-data/labUsers.json" (lines 125-133)